app/js/actions/drug.js

```diff
@@ -1,24 +1,34 @@
 import axios from 'axios';
-import * as actionTypes from './actionTypes';
+import {
+  SEARCH_DRUGS_LOADING,
+  SEARCH_DRUGS_FAILURE,
+  SEARCH_DRUGS_SUCCESS,
+  SELECT_DRUG,
+} from './actionTypes';
 
 const contextPath = window.location.href.split('/')[3];
 const apiBaseUrl = `/${contextPath}/ws/rest/v1`;
 
 export const searchDrugSuccess = drugs => ({
-  type: actionTypes.SEARCH_DRUGS_SUCCESS,
+  type: SEARCH_DRUGS_SUCCESS,
   drugs,
 });
 
 export const searchDrugFailure = error => ({
-  type: actionTypes.SEARCH_DRUGS_FAILURE,
+  type: SEARCH_DRUGS_FAILURE,
   error,
 });
 
 export const searchDrugLoading = status => ({
-  type: actionTypes.SEARCH_DRUGS_LOADING,
+  type: SEARCH_DRUGS_LOADING,
   status,
 });
 
+export const selectDrugSuccess = drugUuid => ({
```

Comment:
Why do you have two methods that look similar? `selectDrugSuccess` one passing drugs while the other drugUuid?

Reply:
selectDrugSuccess action is called when a user selects a drug from the list of drugs, this make the selected drug available in the store for use by everyone. The one passing drugs is the one that makes a list of drugs available to the user based on his/her search parameter. I hope you understand